iOS/Shared/ActivityTimer.swift

```diff
@@ -125,7 +126,50 @@ class ActivityTimer {
                 message += buildRow(row: row)
             }
         }
-        ITMApplication.logger.log(.info, message)
+        ITMApplication.logger.log(.info, "\(title):\n\(message)")
+        if useJSON, logToFile {
+            logToFile(jsonString: message)
+        }
+    }
+
+    func amIBeingDebugged() -> Bool {
```

Comment:
Minor nitpick, I'd prefer something like `currentlyDebugging` or `beingDebugged` rather than `amI...`

Reply:
That actually came from me copying and pasting code from GitHub. I have now renamed the function.